lib/results.js

```diff
@@ -132,22 +133,34 @@ function encodeResult (res, count) {
     var output = '';
     output += (res.ok ? 'ok ' : 'not ok ') + count;
     output += res.name ? ' ' + res.name.toString().replace(/\s+/g, ' ') : '';
-    
+
     if (res.skip) output += ' # SKIP';
     else if (res.todo) output += ' # TODO';
-    
+
     output += '\n';
     if (res.ok) return output;
-    
+
     var outer = '  ';
     var inner = outer + '  ';
     output += outer + '---\n';
     output += inner + 'operator: ' + res.operator + '\n';
-    
+
     if (has(res, 'expected') || has(res, 'actual')) {
-        var ex = inspect(res.expected, {depth: res.objectPrintDepth});
-        var ac = inspect(res.actual, {depth: res.objectPrintDepth});
-        
+        if (process.env.NODE_TAPE_OBJECT_PRINT_DEPTH) {
+            res.objectPrintDepth = process.env.NODE_TAPE_OBJECT_PRINT_DEPTH;
+        }
+        if (typeof res.expected === 'object' &&
+            typeof res.actual   === 'object' &&
+            res.objectPrintDepth             &&
```

Comment:
please do not use spaces for alignment (nor align anything ever, except semantically). There should only be one space around binary operators.

Reply:
ok